app/Http/Controllers/HomeController.php

```diff
@@ -100,6 +100,16 @@ public function showIndex()
 
         $canPageBackward = Incident::notScheduled()->where('created_at', '<', $startDate->format('Y-m-d'))->count() != 0;
 
+        // Determine favicon to use
+        $favicon = 'favicon';
+        if (Component::where('status', '=', 4)->count() > 0 ||
```

Comment:
By default `where` will use the `=` operator, meaning you can just do `Component::where('status', 4)` :)